package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "segment-volume-lambda",
+  "version": "1.0.0",
+  "description": "Pushing volume data from segment into datadog",
+  "scripts": {
+    "depcheck": "./bin/depcheck.sh",
+    "dev": "serverless offline start --config serverless-dev.yml",
+    "process:volumes": "node ./dist/index.js",
+    "lint": "eslint .",
+    "serve": "serverless offline start --config serverless-dev.yml",
+    "test": "jest",
+    "test:integration": "jest integration -c jest.integration.config.js",
+    "typecheck": "tsc --noEmit"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/immediate-media/segment-volume-lambda.git"
+  },
+  "dependencies": {
+    "@datadog/datadog-api-client": "^1.22.0",
+    "axios": "^1.6.7",
+    "dotenv": "^16.4.5"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.23.5",
+    "@babel/plugin-transform-runtime": "^7.23.4",
+    "@babel/preset-env": "^7.23.5",
+    "@commitlint/config-conventional": "^12.1.4",
+    "@types/aws-lambda": "^8.10.130",
+    "@types/jest": "^29.5.10",
+    "@types/node-fetch": "^2.6.9",
+    "@types/original-url": "^1.2.3",
+    "@typescript-eslint/eslint-plugin": "^6.13.2",
+    "@typescript-eslint/parser": "^6.13.2",
+    "commitlint": "^12.1.4",
+    "depcheck": "^1.4.7",
+    "esbuild": "^0.19.8",
+    "eslint": "^8.55.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-import": "^2.29.0",
+    "eslint-plugin-jest": "^27.6.0",
+    "eslint-plugin-jest-formatting": "^3.1.0",
+    "eslint-plugin-json": "^3.1.0",
+    "eslint-plugin-prettier": "^5.0.1",
+    "husky": "^4.2.5",
```

Comment:
This won't be doing anything. It would if we included this bit:

https://github.com/immediate-media/mobile-site-associations/blob/24d7769efbec2e4855fb7fbc1ffbd7f45012aecc/package.json#L21-L25

which basically enforces [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) (which is also what `commitlint` and `@commitlint/config-conventional` are for).